scripts/google_cloud_storage.py

```diff
@@ -24,10 +25,13 @@ def upload_files(
             blob.metadata = {**kwargs}
             blob.upload_from_filename(filename=output_file)
 
-    def get_files(self):
-        output_dir = "outputs"
+    def get_files(self, from_date, output_dir):
 
         for blob in self.client.list_blobs(self.bucket_name):
+            blob_date = blob.name.split("/")[1].split("T")[0]
+
+            if from_date and datetime.strptime(blob_date, "%Y-%m-%d") < from_date:
```

Comment:
Blob object already have timestamps so this can be:
```suggestion
            if from_date and blob.time_created < from_date:
```
However, you will need to make sure `from_date` is time zone aware as you can't compare timezone aware and timezone naive dates.
Like so:
```py
from_date = datetime.now(tz=tzutc()) - timedelta(days=number_of_days)
```
tzutc can be imported using: `from dateutil.tz import tzutc` as it is already part of our dependency.